test/test_request_behavior.py

```diff
@@ -0,0 +1,42 @@
+import os
+import io
+from tempfile import TemporaryFile
+import requests
+from twisted.trial.unittest import TestCase
+from .resources import application
+from twisted.internet import reactor
+from hendrix.deploy.base import HendrixDeploy
+from twisted.internet import threads
+from twisted.internet.defer import ensureDeferred
+import pytest_twisted
+
+@pytest_twisted.inlineCallbacks
+def test_max_upload_bytes():
+    options = {'wsgi': application, 'max_upload_bytes': 200, 'http_port': 9876}
+    deployer = HendrixDeploy(options=options)
+    deployer.addServices()
+    deployer.start()
+
+    def reject_large_uploads():
+
+        # upload 50 bytes, we should be fine.
```

Comment:
What's with the speculation?

```suggestion
        # uploading 50 bytes is fine.
```